repos/rust-lang/rust.toml

```diff
@@ -0,0 +1,54 @@
+org = "rust-lang"
+name = "rust"
+description = "Empowering everyone to build reliable and efficient software."
+bots = ["bors", "rustbot", "rfcbot"]
+
+[access.teams]
+bootstrap = "write"
+cargo = "write"
+clippy = "write"
+compiler = "write"
+infra = "write"
+lang = "write"
+libs = "write"
+libs-api = "write"
+libs-contributors = "write"
+lang-ops = "write"
```

Comment:
lang-ops is not ordered alphabetically

Reply:
For some reason I read this as `libs-ops`, lol. Seriously, one would think that with a PhD I will be able to sort a list of 20 items alphabetically...

Reply:
everybody nows you need 20+ years of experience for that!